docs/basic-usage.ipynb

```diff
@@ -101,29 +101,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import itertools\n",
     "import geopandas\n",
-    "import numpy\n",
     "\n",
-    "south_west = (24.91, 60.19)\n",
-    "north_east = (24.97, 60.24)\n",
+    "# Filepath\n",
+    "fp = \"data/population_points_2020.gpkg\"\n",
```

Comment:
I’m not sure we need to define the file path in a separate line. Would it be enough to straight call `geopandas.read_file("data/population_points_2020.gpkg")`

Reply:
Sure, we can do this